pyroscope/api.go

```diff
@@ -20,7 +20,6 @@ type Config struct {
 	ProfileTypes           []ProfileType
 	DisableGCRuns          bool // this will disable automatic runtime.GC runs between getting the heap profiles
 	DisableAutomaticResets bool // disable automatic profiler reset every 10 seconds. Reset manually by calling Flush method
-	DisableCumulativeMerge bool // disable client side merging of cumulative profiles (alloc_objects, alloc_count, block_count, block_duration, mutex_count, mutex_duration) merging
```

Comment:
This is a breaking change: we can't remove the field. Instead, we should add a [deprecation notice](https://rakyll.org/deprecated/)

Reply:
deprecated